Source/BaseClasses/GDataServiceBase.h

```diff
@@ -21,8 +21,8 @@
   #define GDataUploadFetcherClass GTMSessionUploadFetcher
   #define GDataUploadFetcherClassStr @"GTMSessionUploadFetcher"
 
-  #define kGDataFetcherStatusUnauthorized kGTMSessionFetcherStatusUnauthorized
-  #define kGDataFetcherStatusForbidden kGTMSessionFetcherStatusForbidden
+  #define kGDataFetcherStatusUnauthorized GTMSessionFetcherStatusUnauthorized
+  #define kGDataFetcherStatusForbidden GTMSessionFetcherStatusForbidden
```

Comment:
Wasn't this already landed when https://github.com/google/gdata-objectivec-client/pull/193 was merged in?